src/System.Drawing.Primitives/src/System/Drawing/KnownColor.cs

```diff
@@ -45,9 +46,11 @@ public enum KnownColor
         Window,
         WindowFrame,
         WindowText,
+        */
 
         // "Web" Colors
-        Transparent,
+        FirstColor,
+        Transparent = FirstColor,
```

Comment:
Should we maintain the same enum values as on desktop, or is there no way for these values to escape to user code?